src/content/docs/usage/dev-environments.mdx

```diff
@@ -113,3 +113,48 @@ Out of the box, Daytona can be configured to open your Dev Environment in the fo
 
 Your preferred IDE will be opened automatically and connected to your Dev Environment when using the `daytona open` command.
 
+### Forward Ports from a Dev Environment
```

Comment:
Just noticing that you're referring to `workspaces` as `Dev Environments`.

E.g. `daytona forward <PORT> <DEV_ENVIRONMENT> <PROJECT>` when in the CLI, this command is `daytona forward [PORT] [WORKSPACE] [PROJECT]`.

While I'm not completely against calling them `Dev Environments` (externally), this is something we should discuss with the rest of the team. In the meantime, all references to `Dev Environments` should be swapped to `workspaces/Workspaces` and CLI command "examples" should be synced to the `daytona COMMAND --help` representation.

P.S. It's okay to refer to a workspace as a development environment when explaining what a workspace is but a `Dev Environment` is not a term we use in the project.

Reply:
Pinging @ivan-burazin for input

Reply:
We can rename to Workspace now for clarity to the current users, but have opened an issue to discuss renaming as like @osslate's suggestion: https://github.com/daytonaio/daytona/issues/600

Reply:
Fixed in [57bd99d](https://github.com/daytonaio/docs/pull/18/commits/57bd99d4ee8283f02333c00aa0d85ac5aa99073d).

To keep focus on this PR's purpose of adding port forwarding instructions, I've opened https://github.com/daytonaio/docs/issues/19 to track a docs-wide rename of Dev Environments -> Workspaces.